compiler/sem/sigmatch.nim

```diff
@@ -2311,10 +2311,25 @@ proc paramTypesMatchAux(m: var TCandidate, f, a: PType,
       of tyTyped, tyTypeDesc:
         arg
       of tyStatic:
-        if arg.typ.n.isNil:  # no value on the type
-          argSemantized
-        else:                # value on the type
-          arg.typ.n
+        let n =
+          if arg.typ.n.isNil:  # no value on the type
+            argSemantized
+          else:                # value on the type
+            arg.typ.n
+
+        # XXX: this is super hacky; it's the non-template/macro logic from
+        #      below but in worse
```

Comment:
Sentence fragment?

Not sure what was intended with it, so I can't provide a suggestion.

Reply:
No worries! What I wanted to communicate here is that the handling of implicit conversions is duplicated from the non-template/non-macro path below, but without all the intricacies. I'll reword the comment.

Reply:
Thanks